data.js

```diff
@@ -30,10 +30,10 @@ class Anime {
 }
 
 class MALClient extends AbstractAnimeDataSource {
-    constructor(popuraClient) {
+    constructor(popuraClient, entities) {
         super();
         this._client = popuraClient;
-        this.entities = new Entities();
+        this._entities = entities;
```

Comment:
I think javascript allows us also to do stuff like 
```javascript
get entities() {
    return this._entities
}
```
in case you want to access the entities property (while keeping it unwriteable). It's probably a best practice so we dont have to type a lot of underscores in our code. But we can do that as part of another ticket